octoprint_mrbeam/templates/hard_refresh_overlay.jinja2

```diff
@@ -1,14 +1,24 @@
 <div id="hard_refresh_overlay" class="modal hide fade" data-backdrop="static">
     <div class="modal-header">
-        <h3><i class="icon icon-warning-sign"></i> {{ _('Refresh is needed after the last Software Update') }}</h3>
+        <h3><i class="icon icon-warning-sign"></i> {{ _("A 'Hard Refresh' is needed after the last Software Update") }}</h3>
     </div>
     <div class="modal-body">
         <div class="row-fluid">
             <div class="span5">
                 <div class="image_description">
-                    <p>{{ _("An update has been installed on this Mr Beam. To use the new functions, the user interace must be completely reloaded.") }}</p>
-                    <p> {{ _("We call this a hard refresh. To do this, press the three keys at the same time, depending on your operating system, as shown in the graphic:") }}</p>
-                </div>    
+                    <p>{{ _("An update has been installed on this Mr Beam. To use the new functions, the user interface must be completely reloaded.") }}</p>
+                    <p> {{ _("We call this a 'Hard Refresh'. To do this, press the three keys at the same time, depending on your operating system, as shown in the graphic:") }}</p>
+                    <br>
+                    <ol>
+                        <li>{{ _("Perform the 'Hard Refresh' using the keyboard shortcut") }}</li>
+                        <li>{{ _("Close this Window")}}</li>
+                    </ol>
+                    <br>
+                    <label for="confirm_hard_refresh_checkbox" class="checkbox">
+                        <input type="checkbox" id="confirm_hard_refresh_checkbox" data-bind="checked: hard_refresh_checkbox_checked"/>
+                        <span>{{ _("I completely reloaded the software ('Hard Refresh'). Do not show this window again until the next update.")}}</span>
+                    </label>
```

Comment:
```suggestion
                    <label for="hard_refresh_checkbox"></label>
                    <input type="checkbox" id="hard_refresh_checkbox" data-bind="checked: hardRefreshCheckbox"/>
                    <span>{{ _("I completely reloaded the software ('Hard Refresh'). Do not show this window again until the next update.")}}</span>
                    
```

Reply:
- Please check if this change any styling but the label tag usually does not cover the input tag

Reply:
I need it with the class and inside the label in order to display it side by side. This is already done like this for the safety warning checkboxes

Reply:
If that's the case then let's keep it and I'll add the ID name change separately